sdk/cosmos/azure-cosmos/azure/cosmos/_base.py

```diff
@@ -69,114 +70,104 @@ def GetHeaders(cosmos_client_connection,
     if cosmos_client_connection._useMultipleWriteLocations:
         headers[http_constants.HttpHeaders.AllowTentativeWrites] = "true"
 
-    pre_trigger_include = options.get('preTriggerInclude')
+    pre_trigger_include = options.get("preTriggerInclude")
     if pre_trigger_include:
         headers[http_constants.HttpHeaders.PreTriggerInclude] = (
-            pre_trigger_include
-            if isinstance(pre_trigger_include, str)
-            else (',').join(pre_trigger_include))
+            pre_trigger_include if isinstance(pre_trigger_include, str) else (",").join(pre_trigger_include)
```

Comment:
Not a preference of mine but also not something pylint seems to care about, am intending to scope this PR to only changes related to pylint (applying black formatting being the one exception, because it fixed hundreds of other issues in itself)